doc/source/io.rst

```diff
@@ -1989,6 +1989,46 @@ advanced strategies
 Reading Excel Files
 '''''''''''''''''''
 
+.. versionadded:: 0.17
```

Comment:
ok for now, but maybe make this have sub-sections to make this a bit easier to navigate